src/components/playback/playbackmanager.js

```diff
@@ -1806,12 +1806,18 @@ class PlaybackManager {
                     MediaTypes: 'Audio'
                 });
             } else if (firstItem.MediaType === 'Photo') {
+                const sortOptions = options.sortOptions || {};
+                let sortByValue = options.shuffle ? 'Random' : sortOptions.sortBy;
+                if (sortByValue == null) {
+                    sortByValue = 'SortName';
+                }
```

Comment:
Looking at other sections, can't we use this on each section (with few adjustments)? Does it work?

Reply:
I haven't tried because I do not use it for music, etc... But I can try and check later today also.


Thank you for the feedback!